src/components/Hearing/Comment/index.js

```diff
@@ -478,7 +478,7 @@ class Comment extends React.Component {
         <div className="hearing-comment__comment-wrapper">
           {this.renderCommentHeader(isAdminUser)}
           {!this.props.isReply && this.renderCommentAnswers()}
-          <div className="hearing-comment-body">
+          <div className={`hearing-comment-body ${data.deleted ? "hearing-comment-body-disabled" : ""}`}>
```

Comment:
I would personally use the already imported `classnames` package for conditional classnames instead of doing it manually. Using classnames this line could look like 
`<div className={classnames('hearing-comment-body',{'body-disabled', data.deleted})}>`
which would render as `<div class="hearing-comment-body body-disabled">`